modulefiles/module_gwsetup.gaea.lua

```diff
@@ -15,7 +15,5 @@ load(pathJoin("python", python_ver))
 load("py-jinja2")
 load("py-pyyaml")
 load("py-numpy")
-local git_ver=os.getenv("git_ver") or "2.35.2"
```

Comment:
Are you sure about this?  
I think the ci scripts are using git, and just making sure this is not going to blow up there.

Reply:
Yes, I tested the system `git` version on all platforms.  The only system that needs a non-default version is S4.  Just to be safe, I can revert the changes here and add this to a future PR that needs to be CI tested.  Thoughts?

Reply:
No.  This is good enough for me.  I was just double-checking.